.github/workflows/windows.yml

```diff
@@ -0,0 +1,23 @@
+name: Windows GitHub CI
+
+on: [pull_request, push]
+
+jobs:
+  minimal:
+    runs-on: windows-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: ilammy/msvc-dev-cmd@v1
+    - name: configbuildandtest
+      env:
+        CONFIG_OPTS: no-bulk no-deprecated no-fips no-asm -DOPENSSL_SMALL_FOOTPRINT
+        TARGET: VC-WIN64A
+      run: |
+        New-Item -Path . -Name "_build" -ItemType "directory"
+        Set-Location "_build"
+        perl ..\Configure no-makedepend no-bulk no-deprecated no-fips no-asm -DOPENSSL_SMALL_FOOTPRINT VC-WIN64A
+        perl configdata.pm --dump
+        nmake build_all_generated
+        nmake PERL=no-perl
+        cmd /c nmake test VERBOSE_FAILURE=yes TESTS=-test_fuzz 2>&1
```

Comment:
Why the explicit invokation of cmd? Also, I don't think that redirection is needed here...